issuer-list/src/main/java/com/adyen/checkout/issuerlist/internal/ui/DefaultIssuerListDelegate.kt

```diff
@@ -128,6 +135,13 @@ internal class DefaultIssuerListDelegate<
         _outputDataFlow.tryEmit(outputData)
 
         updateComponentState(outputData)
+
+        val event = GenericEvents.selected(
```

Comment:
Could it be that this method is triggered when something else happens then selecting an issuer?

Reply:
`onInputDataChanged()` gets called only from `updateInputData()` and it gets called only when an issuer is selected.

I know it is not safe to do this here, since the naming is quite confusing, but after we remove `inputData` and change the way we communicate between the delegate and the view, this will be fixed.

Reply:
Yes, I had the same thought. Just wanted to make sure it will not be called at the wrong time 👍